pep-0514.txt

```diff
@@ -158,25 +185,37 @@ The Tag part of the key is intended to uniquely identify an environment within
 those provided by a single company. The key name should be alphanumeric without
 spaces and stable across installations. For example, the Python language
 version, a UUID or a partial/complete hash would be appropriate; an integer
-counter that increases for each new environment may not::
+counter that increases for each new environment may not. It is expected that
+some tools will require users to type the Tag into a command line, and that the
+Company may be optional provided the Tag is unique across all Python
+installations::
 
+    HKEY_CURRENT_USER\Software\Python\ExampleCorp\examplepy
     HKEY_CURRENT_USER\Software\Python\ExampleCorp\3.6
     HKEY_CURRENT_USER\Software\Python\ExampleCorp\6C465E66
 
```

Comment:
Same objection here to opaque IDs like ```6C465E66``` as with the Company. No user's ever going to type ```-p 6C465E66-5A8C-4942-9E6A-D29159480C60/6C465E66```, or even ```-p examplecorp.org/6C465E66``` as an argument to virtualenv.